client/src/resources/lang/en_us.json

```diff
@@ -461,136 +461,143 @@
     "wiki.article.standard.journal.title": "Journal",
-    "wiki.article.standard.journal.text": "The journalist has a journal which is posted publically every morning the journalist chooses to. Everyone knows the journal was written by the journalist so they know the information is trustworthy. The journalist may interview a player each night to gain information to include in the journal.",
+    "wiki.article.standard.journal.text": "The journalist has a journal which is posted publicly every morning the journalist chooses to. Everyone knows the journal was written by the journalist so they know the information is trustworthy. The journalist may interview a player each night to gain information to include in the journal.",
     "wiki.article.standard.forged.title": "Forged",
     "wiki.article.standard.forged.text": "A player who is forged had their alibi and role changed by the forger on their grave.",
     "wiki.article.standard.fullMoon.title": "Full Moon",
-    "wiki.article.standard.fullMoon.text": "Every night is a full moon night except night one and night three. The Werewolf has a different ability depending on weather or not it is a full moon night.",
+    "wiki.article.standard.fullMoon.text": "Every night is a full moon night except night one and night three. The Werewolf has a different ability depending on whether or not it is a full moon night.",
     "wiki.article.standard.vision.title": "Vision",
-    "wiki.article.standard.vision.text": "Visions are the investigative result that a Psychic recieves. Good visions show a list of two players that contains at least one townie. Evil visions show a list of three players that contains at least one non-townie. An Aura can effect a psychic vision.",
+    "wiki.article.standard.vision.text": "Visions are the investigative result that a Psychic receives. Good visions show a list of two players that contains at least one townie. Evil visions show a list of three players that contains at least one non-townie. An Aura can effect a psychic vision. A player with a suspicious aura will not show up in a good vision, and a player with an innocent aura will not show up in an evil vision.",
+    "wiki.article.standard.select.title":"Select",
+    "wiki.article.standard.select.text": "Selecting is when you click the button under a player at night in order to attempt to use your ability on them. Selecting determines who you will visit, before it becomes affected by other effects. By default, you can select a number of players up to as many targets as your role can have. Most roles can not select themselves or dead players, and some roles have further restrictions on selections. Most mafia members and cultists can not select other players on their team.",
```

Comment:
You can select the numbers of players that your role needs to visit to function.

Something like that